com.unity.ml-agents/Runtime/Policies/BehaviorParameters.cs

```diff
@@ -45,9 +45,7 @@ enum BehaviorType
         /// <summary>
         /// The team ID for this behavior.
         /// </summary>
-        [HideInInspector]
-        [SerializeField]
-        [FormerlySerializedAs("m_TeamID")]
+        [HideInInspector, SerializeField, FormerlySerializedAs("m_TeamID")]
```

Comment:
You can do that ?!
Should we try to be consistent and make it the same way everywhere?

Reply:
I did a simple find-and-replace for the ones for HideInInspector+ SerializeField+ FormerlySerializedAs. That's as much effort as I want to put into it for this PR.